packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -49,14 +50,16 @@ enum Tabs {
 }
 
 const createDraftInvoiceLine = (
-  itemId: string,
+  item: ItemRowFragment,
```

Comment:
`DraftInboundLineFragment` had the intention of creating drafts more easily. Turns out it basically had all of the fields of the `InboundLineFragment` anyway so I removed it - extending `DraftInboundLine` from `InboundLineFragment`. This meant I needed to add these fields